main.nf

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env nextflow
 /*
 ========================================================================================
-                         nf-core/cageseq
+                                    nf-core/cageseq
 ========================================================================================
- nf-core/cageseq Analysis Pipeline.
- #### Homepage / Documentation
- https://github.com/nf-core/cageseq
+nf-core/cageseq Analysis Pipeline.
+#### Homepage / Documentation
+https://github.com/nf-core/cageseq
```

Comment:
shouldn't we use nf-co.re/cageseq?

Reply:
or list both?

Reply:
Yes, maybe we should update the template for that (which is the source for this).